### DISCUSSION TITLE
a few features needed

### DISCUSSION THREAD

**Discussion by hiqsociety** [vlang/v#12706] [Ideas]
1. reverse proxy example for pico
2. redis example like redcon https://github.com/tidwall/redcon

**Comment by JalonSolov** [comment]
For 2, have you looked at https://github.com/patrickpissurno/vredis ?

**Comment by hiqsociety** [comment]
not talking about redis client but redis server. e.g.
https://github.com/IceFireDB/redhub
https://github.com/tidwall/redcon

**Comment by JalonSolov** [comment]
If you need an in-memory database, there is https://github.com/elliotchance/vsql - pure V.

**Comment by hiqsociety** [comment]
the benchmark of it is really terrible. i hope to write my own redis server using v.
